apps/site/assets/ts/stop/components/StopPageOverlay.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement, useState } from "react";
+import { Route, Stop } from "../../__v3api";
+import Modal from "../../components/Modal";
+import { ScheduleWithTimestamp } from "../../models/schedules";
+
+interface StopPageOverlayProps {
+  routes: Route[];
+  stop: Stop;
+  schedules: ScheduleWithTimestamp[];
+}
+
+const StopPageOverlay = ({
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  routes,
+  stop,
+  schedules
+}: /* eslint-enable @typescript-eslint/no-unused-vars */
+StopPageOverlayProps): ReactElement<HTMLElement> => {
+  const [modalOpen, toggleModal] = useState(false);
+
+  const modalClick: () => void = () =>
+    modalOpen ? toggleModal(false) : toggleModal(true);
+
+  return (
+    <div className="sm-up-hide">
+      <button type="button" className="overlay-open" onClick={modalClick}>
+        open overlay
+      </button>
+      {modalOpen && (
+        <Modal
+          className="m-stop-routes-and-map"
```

Comment:
question (non-blocking): Is it intentional that this class name is the same as what is on StopPageRedesign?

Reply:
Yes, since this was going to be the mobile view. But non plural `route` is probably a better name here.